src/main/java/org/spongepowered/common/mixin/core/item/MixinItemFishingRod.java

```diff
@@ -54,10 +54,12 @@ public ItemStack onItemRightClick(ItemStack itemStack, World world, EntityPlayer
             player.fishEntity.handleHookRetraction();
         } else {
             EntityFishHook fishHook = new EntityFishHook(world, player);
-            if (!Sponge.getGame().getEventManager()
+            if (world.isRemote || !Sponge.getGame().getEventManager()
```

Comment:
Except, again, its un-needed to have two more for events.

Really, by default it is DEFAULT which means "I don't give a flip about what side I get ran against, just run this code". Otherwise, I can choose to only run on the CLIENT or the SERVER.